modules/controller/rpc-controller.js

```diff
@@ -828,7 +868,17 @@ class RpcController {
                     },
                 );
             })
-            .catch(e => {
+            .catch((e) => {
+                Models.handler_ids.update(
```

Comment:
Let's create new method updateFailedHandlerId and call it in catch clauses so that we don't repeat this Models.update code

Reply:
Resolved with [this commit](https://github.com/OriginTrail/ot-node/pull/1738/commits/91d69a875ff201644671d6ddd3bc62ec1d939e4b)